model_compression_toolkit/core/common/mixed_precision/resource_utilization_tools/ru_methods.py

```diff
@@ -87,6 +90,88 @@ def weights_size_utilization(mp_cfg: List[int],
     return np.array(weights_memory)
 
 
+def calc_graph_cuts(graph: Graph) -> List[Cut]:
+    """
+    Calculate graph activation cuts.
+    Args:
+        graph: A graph object to calculate activation cuts on.
+
+    Returns:
+        A list of activation cuts.
+
+    """
+    memory_graph = MemoryGraph(deepcopy(graph))  # TODO maxcut: do we need the deepcopy?
+    _, _, cuts = compute_graph_max_cut(memory_graph)
+
+    if cuts is None:
+        return None
+    else:
+        # filter empty cuts and cuta that contain nodes with activation quantization disabled.
+        filtered_cuts = []
+        for cut in cuts:
+            if len(cut.mem_elements.elements) > 0 and any(
```

Comment:
consider assgining the condition to a variable and make the if condition clearer